aiida/cmdline/params/types/identifier.py

```diff
@@ -46,24 +47,29 @@ def __init__(self, sub_classes=None):
             will be mapped upon. These classes have to be strict sub classes of the base orm class defined
             by the orm class loader
         """
-        from aiida.common import exceptions
+        if sub_classes is not None and not isinstance(sub_classes, tuple):
```

Comment:
Can you update the constructor docstring please. It currently still states that the entry points are checked to exist in the constructor, but that is no longer true.

Reply:
Done.